### PR TITLE
add section

### PR DESCRIPTION
**Pull request by ardyanandakirana**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,6 +12,79 @@
     />
   </head>
   <body>
+
+    <div class="main-section pt-md-5" style="background-color: #f1f3ff">
```
Comment (CoreX20) [inline index.html]: change background color to : #e5e5e5

**Review by CoreX20 — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -12,6 +12,79 @@
...
+      <div class="container h-100 d-flex pt-md-5">
+        <div class="row d-flex justify-content-center align-items-center">
+          <div class="col-md-6 tagline py-5">
+            <h2>Sewa & Rental Mobil Terbaik di kawasan (Lokasimu)</h2>
+            <p>
+              Selamat datang di Binar Car Rental. Kami menyediakan mobil
+              kualitas terbaik dengan harga terjangkau. Selalu siap melayani
+              kebutuhanmu untuk sewa mobil selama 24 jam.
+            </p>
+            <a href="#" class="btn btn-success">Mulai Sewa Mobil</a>
```
Comment (amiqbal7) [inline index.html]: change button color to primary

**Review by amiqbal7 — COMMENTED** [review]
change button color

File: index.html (hunk 1 above)
Comment (ardyanandakirana) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (ardyanandakirana) [reply index.html]: done

**Review by CoreX20 — APPROVED** [review]

**Pull request merged**